resources/views/settings/ldap.blade.php

```diff
@@ -275,11 +215,7 @@
                                 {{ Form::label('ldap_fname_field', trans('admin/settings/general.ldap_fname_field')) }}
                             </div>
                             <div class="col-md-9">
-                                @if (config('app.lock_passwords')===true)
```

Comment:
Why were the lock_password calls removed here?

Reply:
I moved the lock_passwords to a variable in the controller. A little cleaner than having all the if statements. $setting->demoMode returns disabled if the site is in demo mode.